### PR TITLE
Add routing and router views

### PR DESCRIPTION
**Pull request by gordon-cheung**

### CODE REVIEW

File: frontend/src/router.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export default new Router({
+    routes: [
+        {
+            path: "/",
+            name: "home",
+            component: Home
+        },
+        {
+            path: "/lobby",
+            redirect: "/"
```
Comment (bseto) [inline frontend/src/router.js:17]: does this mean route back to /, meaning route to "home"?

File: frontend/src/views/Home.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<template>
```
Comment (bseto) [inline frontend/src/views/Home.vue:1]: What is home.vue?

File: frontend/src/webSocket.js [hunk 3]
```diff
@@ -32,6 +32,9 @@ export const ArcadeWebSocket = new Vue({
             } else {
                 console.error("NOT CONNECTED");
             }
+        },
+        isConnected: function() {
+            return webSocket.readyState === WebSocket.OPEN;
```
Comment (bseto) [inline frontend/src/webSocket.js:37]: nice

File: frontend/src/router.js (hunk 1 above)
Comment (gordon-cheung) [reply frontend/src/router.js:17]: Yes

File: frontend/src/views/Home.vue (hunk 2 above)
Comment (gordon-cheung) [reply frontend/src/views/Home.vue:1]: The join and chat buttons

File: frontend/src/webSocket.js (hunk 3 above)
Comment (gordon-cheung) [reply frontend/src/webSocket.js:37]: vgud

**Pull request merged**